UPGRADE-2.8.md

```diff
@@ -399,3 +399,23 @@ FrameworkBundle
        session:
            cookie_httponly: false
    ```
+
+Config
+------
```

Comment:
the UPGRADE file is about showing required/recommended changes because of BC breaks of deprecations.

this is just adding a new feature in a BC way, so a line in the changelog would be enough. Adding it in the UPGRADE file just makes the file less focused